cob_bms_driver/src/fake_bms.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import rospy
+from cob_msgs.msg import PowerState
+from cob_srvs.srv import SetFloat, SetFloatResponse
+from std_srvs.srv import SetBool, SetBoolResponse
+
+class FakeBMS(object):
+    def __init__(self):
+        self.srv_charging = rospy.Service('~set_charging', SetBool, self.charging_cb)
+        self.srv_relative_remaining_capacity = rospy.Service('~set_relative_remaining_capacity', SetFloat, self.relative_remaining_capacity_cb)
+        self.pub_power_state = rospy.Publisher('/power_state', PowerState, queue_size = 1)
```

Comment:
I thought, we discussed to publish 6 separate topics `voltage`, `current`, `remaining_capacity`, `full_charge_capacity`, `temperature`, `battery_charging` - as the real bms driver does - instead of publishing the aggregated `/power_state`
Because then we can use the same `power_state_aggregator.py` node to aggregate the `/power_state` topic in simulation and in real hw!